package.json

```diff
@@ -46,7 +46,7 @@
     "@vue/cli-plugin-router": "~4.5.0",
     "@vue/cli-plugin-typescript": "~4.5.0",
     "@vue/cli-plugin-unit-jest": "~4.5.0",
-    "@vue/cli-service": "~4.5.0",
+    "@vue/cli-service": "4.5.16",
```

Comment:
I think this is unwanted change. Please revert

Reply:
Sir, I have done it.